app/templates/components/ecosystem-icon.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="ecosystem-icon">
+  <svg class="background" xmlns="http://www.w3.org/2000/svg" width="133" height="130"><defs/><path fill={{@bg}} fill-rule="nonzero" d="M.91 27.535l15.047 93.176c.878 5.44 5.993 9.145 11.436 8.283l96.19-15.233c5.45-.863 9.172-5.98 8.314-11.43L117.264 9.301c-.857-5.451-5.969-9.178-11.42-8.326L9.238 16.06a10 10 0 00-8.33 11.475z"/></svg>
+  <img class="logo {{if @rotate "rotate"}}" src="/images/home/logos/{{@icon}}.svg" alt="">
```

Comment:
what is the SVG doing in this case? 

the image needs valid alt text. Please turn on ember-template-lint if it is not already turned on- code shouldn't be making it this far with these type of basic errors.

Reply:
So the SVG is the background, it's an inline SVG so that we can pass the background fill colour in via the `{{@bg}}` attribute

There is an `alt=""` attribute so that's why it's not causing an error with the template lint. I thought that this is what you're supposed to do when something is presentational 😞I'm sorry for the mistake, I'll try to do better in future. I have made the alt be the same as the icon now. 

Also I've made the changes over in this other PR because I'm splitting this one: https://github.com/ember-learn/ember-website/pull/529